### PR TITLE
fix(Scripts/Commands): cs_list commands

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10477

## Tests Performed:

- Not-Tested
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Kitzunu — CHANGES_REQUESTED** [review]
should probably check for id2 and id3 as well :)

**Review by Nefertumm — APPROVED** [review]

**Comment by Nyeriah** [comment]
Hey, any luck with this? You can't teleport to creatures without this command at all, very difficult

**Comment by IntelligentQuantum** [comment]
> Hey, any luck with this? You can't teleport to creatures without this command at all, very difficult

.list crea xxx - works fine, for other ids idk how can i do it.

**Comment by Nyeriah** [comment]
> > Hey, any luck with this? You can't teleport to creatures without this command at all, very difficult
> 
> .list crea xxx - works fine, for other ids idk how can i do it.

change the query so it's

SELECT ... FROM creature WHERE id1 = XX OR Id2 = XX OR id3 = XX

**Review by Kitzunu — APPROVED** [review]

**Comment by Nyeriah** [comment]
Conflicts here

File: src/server/scripts/Commands/cs_list.cpp [hunk 1]
```diff
@@ -85,19 +85,19 @@ class list_commandscript : public CommandScript
...
+            result = WorldDatabase.Query("SELECT guid, position_x, position_y, position_z, map, (POW(position_x - '{}', 2) + POW(position_y - '{}', 2) + POW(position_z - '{}', 2)) AS order_ FROM creature WHERE id1='{}' OR id2='{}' OR id3='{}' ORDER BY order_ ASC LIMIT {}",
+                                          player->GetPositionX(), player->GetPositionY(), player->GetPositionZ(), uint32(creatureId), uint32(creatureId), uint32(creatureId)count);
         }
         else
-            result = WorldDatabase.PQuery("SELECT guid, position_x, position_y, position_z, map FROM creature WHERE id1='%u' OR id2='%u' OR id3='%u' LIMIT %u",
-                                          uint32(creatureId), uint32(creatureId), uint32(creatureId), count);
+            result = WorldDatabase.Query("SELECT guid, position_x, position_y, position_z, map FROM creature WHERE id1='{}' OR id2='{}' OR id3='{}' LIMIT {}",
+                                          uint32(creatureId), uint32(creatureId), uint32(creatureId)count);
```
Comment (Kitzunu) [inline src/server/scripts/Commands/cs_list.cpp]:
```suggestion
                                          uint32(creatureId), uint32(creatureId), uint32(creatureId), count);
```

**Pull request merged**